cost-analyzer/templates/cost-analyzer-cluster-role-template.yaml

```diff
@@ -55,6 +55,15 @@ rules:
       - get
       - list
       - watch
+{{- $isLeaderFollowerEnabled := include "cost-analyzer.leaderFollowerEnabled" . }}
+{{- if $isLeaderFollowerEnabled }}
+  - apiGroups:
+      - coordination.k8s.io
+    resources:
+      - leases
+    verbs:
+      - '*'
+{{- end }}
```

Comment:
Way late to the party here. Should we only have these permissions in the Kubecost install namespace, rather than the whole cluster? My skim of the documentation suggests that leases are namespaced.